style.css

```diff
@@ -65,105 +169,147 @@ nav img {
-}
+  hr {
+    size: 1px;
+    color: #1c1a19;
+  }
 
-aside img {
-  margin: 1rem;
-}
+  main li {
+    background: url("media/art.png");
+    background-repeat: no-repeat;
+    background-size: cover;
+    background-position: center center;
+    padding-top: 7.2rem;
+    margin-top: 1rem;
+  }
 
-nav li img {
-  height: 2rem;
-}
+  .grid-container {
+    display: grid;
+    grid-template-rows: auto auto auto auto auto auto;
+    row-gap: 0.1rem;
+    align-items: center;
+    justify-content: center;
+  }
 
-#logo {
-  font-size: 2rem;
-  font-weight: bolder;
-}
+  main img {
+    width: 100%;
+  }
 
-a {
-  text-decoration: none;
-  color: #fff;
-}
+  main article {
+    text-align: center;
+  }
 
-nav a:hover {
-  background-color: #fff5e1;
-  border: solid #f55800;
-  padding: 5px;
-  color: #f55800;
-}
+  .text {
+    font-size: 1.6rem;
+    font-weight: 400;
+    margin: 2rem;
+  }
 
-img:hover {
-  border: solid #f55800;
-  background-color: #fff5e1;
-  padding: 5px;
-}
+  .grid-item h2 {
+    margin: 2rem;
+  }
 
-section {
-  background-image: url(media/Illustration-header.png);
-  background-color: #1c1a19;
-  background-attachment: fixed;
-  background-repeat: no-repeat;
-  background-size: 40%;
-  height: 25%;
-  max-width: 100%;
-  background-position: right;
-  display: flex;
-  padding: 10% 0 20% 0;
-}
+  #work-1 {
+    color: #1c1a19;
+  }
 
-aside ul {
-  display: flex;
-  flex-direction: column;
-  margin: 2rem 7rem 0 2rem;
-  align-self: center;
-  list-style: none;
-}
+  button {
+    padding: 1rem 1.2rem;
+    font-weight: 500;
+    margin: 1.2rem;
+    border: none;
+    background-color: inherit;
+  }
 
-h1 {
-  color: #f55800;
-  font-family: 'Crete Round', sans-serif;
-  font-size: 600%;
-  margin-bottom: 0%;
-}
+  .orange-button {
+    display: block;
+    background-color: #ff6b00;
+    padding: 1.2rem;
+    margin-left: 2.4rem;
+    font-weight: 700;
+    width: inherit;
+    color: #fff;
+  }
 
-h2 {
-  font-weight: 50rem;
-  font-size: 3.2rem;
-}
+  article button {
+    border: solid #8993a4;
+  }
 
-div {
-  width: 45%;
-}
+  .btn {
+    display: flex;
+    justify-content: space-around;
+    color: #1c1a19;
+  }
 
-p {
-  font-size: 200%;
-  line-height: 150%;
-}
+  li .orange-button {
+    width: 100%;
+    margin: 0;
+  }
 
-aside {
-  margin-top: 10em;
+  li button {
+    background: rgba(255, 255, 255, 0.24);
+    color: #fff;
+  }
+
+  /* STyling the About me section */
+  #about {
+    color: #fff;
+    background-color: #1c1a19;
+  }
+
+  #intro {
+    background-image: url("media/about-me.png");
+    background-repeat: no-repeat;
+    background-position: right;
+    background-position-y: bottom;
+    margin-top: 10rem;
+    padding-top: 12rem;
+    padding-bottom: 22rem;
+  }
+
+  #about hr {
+    color: #fff;
+    margin: 0;
+  }
+
+  #cv {
+    display: grid;
+    grid-template-rows: auto auto auto;
+    justify-content: center;
+    text-align: center;
+    list-style: none;
+  }
+
```

Comment:
- [ ] Kindly use `flexbox` here to better align the contents, as it is also mentioned in the requirements:

> Make sure the content of the "About myself" section is aligned with flexbox.